docs/howto/how_to_leverage_scenario_editor.md

```diff
@@ -96,6 +93,33 @@ Once you have placed the Ego car's end point on the map, The end point coordinat
 
 Finally, you can always come back and edit the existing attributes of the ego car by clicking on the ego car on the map. This will open its attributes tab in the right-hand attributes window.
 
+#### Participants' Configuration
+
+If you select `Participant` from the configuration menu, you can place your obstacle in your scenario by clicking on a desired location on the map. You will notice that your mouse pointer will turn into a cross until you place the new obstacle on your map. Once you place it, a form will appear on the right-hand attributes window as it did with `Ego Car`. Before you edit the fields on the form, you can change the position of the obstacle by clicking and dragging it. You can also modify its heading by clicking on the arrow head. Once you have finalized the heading and position of your participant, you can start working on specific details mentioned in the form - type, length, speed and motion type. 
+
+![](images/obstacle.png)
+
+In the Basic Information section, you will notice an auto-generated ID along with a description textbox. You could give your obstacle a suitable ID as well as a description about its expected behavior. You could also specify what is the type of the obstacle, which will be set to `Car` by default. Upon selecting a different type, the obstacle on your screen will change accordingly. You will also need to determine its initial speed and other attributes including width, length and height. There are predetermined values for each vehicle type, which can be changed. 
+
+In the Initial State section, you will need to set the speed of the obstacle which can be either set in `m/s` or `km/hr`. The coordinates and heading of the obstacle is preset and can be changed by directly editing the obstacle's position on the map.
+
+In Runtime Configuration, you can set whether the participant is mobile or static. Should you select static, you have finished setting up your obstacle and are ready to save. If you select mobile instead, you would need to set its `Trigger Type`.
+
+Once you have completed your setup, click on the `add route point` button to set the obstacle's trajectory points as seen in the image below. 
```

Comment:
this configuration should be part of the 'mobile' participant only.

Reply:
Agreed.. I changed the alignment so that it appears in that way.